db/migrate/20170215190153_add_description_to_authorization_documents.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToAuthorizationDocuments < ActiveRecord::Migration
+  def change
+    add_column :authorization_documents, :description, :integer, default: 0
```

Comment:
The `authorization_documents`' default value should be `bank_authorization`?
`default: 0` will set the first element of the array.